package-lock.json

```diff
@@ -10,7 +10,8 @@
       "license": "GPL-3.0-only",
       "dependencies": {
         "electron-squirrel-startup": "^1.0.1",
-        "node": "^18.20.3"
+        "node": "^18.20.3",
+        "wdio": "^6.0.1"
```

Comment:
why did you add wdio as dependency? should be only for dev testing, not building the application for final user